### DISCUSSION TITLE
Assigning Intune Mobile Apps Quickly and Consistently - odds+endpoints

### DISCUSSION THREAD

**Discussion by giscus** [ennnbeee/oddsandendpoints-web#10] [Comments]
# Assigning Intune Mobile Apps Quickly and Consistently

Automatically assigning Android Enterprise Google Play apps and iOS iPhone and iPad mobile apps in Microsoft Intune using PowerShell and Graph API.

https://memv.ennbee.uk/posts/assigning-intune-applications/

**Comment by addams45** [comment]
hello , i exactly search this function  for my job, thanks a lot, but after testing it seems an issue when we i try to discover application all i blank, no choice but i have more than 100 apps register in my intune android and ios store, any idea ?

**Reply by ennnbeee** [reply]
Ahh thanks, looks like the scope permissions were incorrect. I've updated them to include the required scope `DeviceManagementApps.ReadWrite.All`, and updated the script. Let me know if you have any more issues.
